### ISSUE TITLE
Mapping already exist on startup

### ISSUE THREAD

**Issue by metalelf0** [kkoomen/vim-doge#10]
**Describe the bug**

When I start nvim, I get the following output:
```
Error detected while processing /Users/metalelf0/.config/nvim/plugged/vim-doge/plugin/doge.vim:                                                                              
line   99:                                                                                                                                                                   
E227: mapping already exists for ^I                                                                                                                                          
line  100:                                                                                                                                                                   
E227: mapping already exists for <80>kB                                                                                                                                      
line  101:                                                                                                                                                                   
E227: mapping already exists for ^I  
```

However none of this mappings seem overridable with a configuration. What are they used for?

As a rule of thumb every plugin-exposed mapping should be overridable by the user via custom configuration.

Thanks and keep up the great work!

**Comment by kkoomen** [comment]
Hi!

I had this one on my list already! Will fix it in 4 hours from now.

**Comment by kkoomen** [comment]
> However none of this mappings seem overridable with a configuration. What are they used for?
>
>As a rule of thumb every plugin-exposed mapping should be overridable by the user via custom >configuration.

These are the mapping defined [here](https://github.com/kkoomen/vim-doge#options). The ones being mapped already are the jump triggers and the backspace.

`<80>kB` = backspace
`^I` = Tab

**Comment by kkoomen** [comment]
This has been fixed in [v1.4.2](https://github.com/kkoomen/vim-doge/tree/v1.4.2). Thanks for submitting your issue. If you experience more issues, feel free to submit them here at the issue tracker.

**Comment by metalelf0** [comment]
Thank you! :)
